README.md

```diff
@@ -156,8 +156,28 @@ $ open ./android/app/build/outputs/apk
 
 [React native docs for IOS](http://facebook.github.io/react-native/releases/0.49/docs/running-on-device.html#building-your-app-for-production)
 
+## Troubleshooting
+If you got `Execution failed for task ':app:compileDebugAidl'` when running `yarn android`, try to update Android Gradle plugin to version 3.1.1 and Gradle to version 4.4.
+
+Here is how to do that:
+1. Open the project in **Android Studio**
+1. Click `Update`
```

Comment:
hey @haruelrovix thanks for the PR, could you specify here "where" they should click "Update", I think it might be a bit unclear for people, especially if they don't know Android Studio.

Many thanks :)

Reply:
@orYoffe updated. 👌